app/org/sagebionetworks/bridge/services/UserProfileServiceImpl.java

```diff
@@ -22,14 +25,17 @@
     
     private SendMailService sendMailService;
 
+    @Autowired
     public void setAccountDao(AccountDao accountDao) {
         this.accountDao = accountDao;
     }
     
+    @Autowired
     public void setExecutorService(ExecutorService executorService) {
         this.executorService = executorService;
     }
```

Comment:
This raises the more general point: @Resource(name="foo") is always less
ambiguous than @Autowired, should we switch to naming all components and
using annotations that take component names instead of @Autowired?

On Mon, Mar 2, 2015 at 2:00 PM, Dwayne Jeng <notifications@github.com>
wrote:

> In app/org/sagebionetworks/bridge/services/UserProfileServiceImpl.java
> <https://github.com/Sage-Bionetworks/BridgePF/pull/513#discussion_r25642872>
> :
>
> >      public void setAccountDao(AccountDao accountDao) {
> >          this.accountDao = accountDao;
> >      }
> >
> > +    @Autowired
> >      public void setExecutorService(ExecutorService executorService) {
> >          this.executorService = executorService;
> >      }
>
> We don't currently have more than one ExecutorService, but we might.
> Consider changing this to
> @Resource <https://github.com/Resource>(name = "asyncExecutorService")
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Sage-Bionetworks/BridgePF/pull/513/files#r25642872>.
>